### PR TITLE
Update competency_questions.md

### PR DESCRIPTION
**Pull request by Zack-83**

### CODE REVIEW

File: docs/competency_questions.md [hunk 1]
```diff
@@ -4,20 +4,29 @@ Competency questions can be understood as the test cases with which one can chec
...
+ * What is the dimensionality of the experiment (e.g. single spectra, line maps, surface maps, volume scans...)?
 
- * What was the assayed sample, including its relevant attributes?
+ * What kind of post-processing (e.g. smoothing, baseline subtraction, band integration, peak recognition, image reconstruction) was done?
 
- * What assay outputs were produced, such as what spectra (e.g. different formats)?
+### Questions addressing the full database
 
- * What kind of pre- and post-processing was done?
+* Find me all Raman spectra of wolfram wires on a silicium substrate covering the range 600-1800 cm^-1.
```
Comment (Zack-83) [inline docs/competency_questions.md:31]:
```suggestion

* Find me which is a safe range of incoming laser powers to perform Raman experiments at wavelength Z on probe XY.

```

File: docs/competency_questions.md [hunk 2]
```diff
@@ -4,20 +4,31 @@ Competency questions can be understood as the test cases with which one can chec
...
+ * What kind of post-processing (e.g. smoothing, baseline subtraction, band integration, peak recognition, image reconstruction) was done?
 
- * What was the assayed sample, including its relevant attributes?
+### Questions addressing the full database
 
- * What assay outputs were produced, such as what spectra (e.g. different formats)?
+* Find me all Raman spectra of wolfram wires on a silicium substrate covering the range 600-1800 cm^-1.
 
- * What kind of pre- and post-processing was done?
+* Find me which is a safe range of incoming laser powers to perform Raman experiments at wavelength Z on probe XY.
```
Comment (Zack-83) [inline docs/competency_questions.md:33]:
```suggestion

* Find me a suitable solvent which is compatible with sample XY, i.e. which does not have any Raman peak or band between wavenumbers W1 and W2.

```

File: docs/competency_questions.md [hunk 3]
```diff
@@ -4,20 +4,29 @@ Competency questions can be understood as the test cases with which one can chec
 
 As previously mentioned, VIBSO's scope is the semantic description of vibrational spectroscopy research data. So far we have thus identified the following competency questions:
 
- * What kind of vibrational spectroscopy was performed?
+### Questions addressing one single data set
+
+ * What kind of vibrational spectroscopy was performed (as detailed as possible: enhanced, resonant, stimulated, coherent, anti-Stokes...)?
 
  * In what context was it performed, e.g. as part of what study or investigation?
+> Requires to clarify the hierarchy in the ontology: e.g. experiment < investigation < project
```
[the remark this answers is not in the document]
Comment (StroemPhi) [reply docs/competency_questions.md:12]: But to answer it here, we do import OBI:0000011 (planned process) which is mapped to ISAs investigation and OBI:0000066 (investigation) is mapped to ISAs study (see our [Raman spectroscopy T-Box.drawio](https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/blob/main/docs/images/Raman%20spectroscopy%20T-Box.drawio)). So with VIBSO we could currently state that an example Ramans spectroscopy assay is the part of an anonymous ISA study (OBI:0000066), which is part of a specific DGF funded project (ISA investigation / OBI planned process) and which has an information content entity as output that is about the assay :


```
example_db:RamanSpecAssay1  rdf:type      CHMO:0000663 (confocal Raman spectroscopy) ;
                            BFO:0000050 (part of)       [ rdf:type     OBI:0000066 ;
                                                          BFO:0000050 https://gepris.dfg.de/gepris/projekt/247890164 ;
                                                          OBI:0000299 (has_specified_output)       https://doi.org/10.1002/jrs.6482 ] .            
https://doi.org/10.1002/jrs.6482   IAO:0000136 (is about)   example_db:RamanSpecAssay1.
```

**Review by StroemPhi — CHANGES_REQUESTED** [review]
The comments about the competency questions should be deleted from the docs.

File: docs/competency_questions.md [hunk 4]
```diff
@@ -4,20 +4,29 @@ Competency questions can be understood as the test cases with which one can chec
...
 
- * What protocol and devices were used?
+ ~* What protocol and devices were used?~
+> Somehow redundant with the following one; "what protocol" meaningful only within a workgroup
```
[the remark this answers is not in the document]
Comment (StroemPhi) [reply docs/competency_questions.md:15]: So I guess this question should best be changed just to: "What devices were used?"

File: docs/competency_questions.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Zack-83) [reply docs/competency_questions.md:12]:
```suggestion
```

File: docs/competency_questions.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Zack-83) [reply docs/competency_questions.md:15]:
```suggestion
```

File: docs/competency_questions.md [hunk 5]
```diff
@@ -4,20 +4,33 @@ Competency questions can be understood as the test cases with which one can chec
...
- * What kind of vibrational spectroscopy was performed?
+### Questions addressing one single data set
+
+ * What kind of vibrational spectroscopy was performed (as detailed as possible: enhanced, resonant, stimulated, coherent, anti-Stokes...)?
 
  * In what context was it performed, e.g. as part of what study or investigation?
+> Requires to clarify the hierarchy in the ontology: e.g. experiment < investigation < project
 
- * What protocol and devices were used?
+ ~* What protocol and devices were used?~
```
Comment (Zack-83) [inline docs/competency_questions.md:14]:
```suggestion
 * What devices were used?
```

File: docs/competency_questions.md [hunk 6]
```diff
@@ -4,20 +4,33 @@ Competency questions can be understood as the test cases with which one can chec
...
+> Somehow redundant with the following one; "what protocol" meaningful only within a workgroup
 
  * What were the protocol parameters & device configurations?
 
- * Who was involved in the assay, e.g. PI, PhD, lab assistant etc.? (identified by a PID like ORCID at best)
+ * Who was involved in the assay? (identified by a PID like ORCID at best)
+ > It is more probable that the users document the names of the co-authors than the academic degree / role
```
Comment (Zack-83) [inline docs/competency_questions.md:20]:
```suggestion
```

**Review by StroemPhi — APPROVED** [review]

**Pull request merged**